tests/calendar-date-overrides.test.ts

```diff
@@ -266,20 +267,21 @@ describe('Testing national calendar overrides', () => {
     });
   });
 
-  describe('Saint Ladislaus', () => {
-    test('A feast in Hungary but an optional memorial in Slovakia', async () => {
-      const hungaryDates = Object.values(
-        await new Romcal({ localizedCalendar: Hungary_En }).generateCalendar(2018),
+  describe('Saint Oliver Plunket', () => {
+    test('A memorial in Ireland but an optional memorial in England', async () => {
+      const irelandDates = Object.values(
+        await new Romcal({ localizedCalendar: Ireland_En }).generateCalendar(2019),
       ).flat();
-      const slovakiaDates = Object.values(
-        await new Romcal({ localizedCalendar: Slovakia_Sk }).generateCalendar(2018),
+      const englandDates = Object.values(
+        await new Romcal({ localizedCalendar: England_En }).generateCalendar(2019),
       ).flat();
-      const ladislausIOfHungaryHungary = hungaryDates.find((d) => d.key === 'ladislaus_i_of_hungary');
-      const ladislausIOfHungarySlovakia = slovakiaDates.find((d) => d.key === 'ladislaus_i_of_hungary');
-      expect(ladislausIOfHungaryHungary?.precedence).toEqual(Precedences.ProperFeast_8f);
-      expect(ladislausIOfHungarySlovakia?.precedence).toEqual(Precedences.OptionalMemorial_12);
-    });
-  });
+      const oliverPlunketBishopIreland = irelandDates.find((d) => d.key === 'oliver_plunket_bishop');
+      const oliverPlunketBishopEngland = englandDates.find((d) => d.key === 'oliver_plunket_bishop');
+
+      expect(oliverPlunketBishopIreland?.precedence).toEqual(Precedences.ProperMemorial_11b);
+      expect(oliverPlunketBishopEngland?.precedence).toEqual(Precedences.OptionalMemorial_12);
+    })
+  })
```

Comment:
@emagnier, I had to update this test, as there is no celebration of Saint Ladislaus inscribed in the national calendar of Slovakia.

I chose Saint Oliver Plunket, as that celebration was the first one I found that was not celebrated in the GRC, but celebrated in two particular calendars with different rank.